app/js/actions/orderAction.js

```diff
@@ -1,11 +1,16 @@
-import { SET_ORDER_ACTION } from './actionTypes';
+import * as types from './actionTypes';
```

Comment:
We generally do not recommend using wildcard imports https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports

Reply:
noted! will revise

Reply:
fixed!